Languages/en.ini

```diff
@@ -926,6 +928,10 @@ Dark=Dark
 Darker=Darker
 Darkest=Darkest
 
+# FaPlus
+Enabled=Enabled

```

Comment:
Please find a better name.

Reply:
Renamed to `ShowFaPlusWindow=Display FA+ Window`